src/nbl/CMakeLists.txt

```diff
@@ -116,7 +116,7 @@ nbl_get_conf_dir(NABLA_CONF_DIR_DEBUG Debug)
 nbl_get_conf_dir(NABLA_CONF_DIR_RELEASE Release)
 if(NBL_EMBED_BUILTIN_RESOURCES)	
 	add_subdirectory(builtin)
-	NBL_ADD_BUILTIN_RESOURCES()
+	NBL_ADD_BUILTIN_RESOURCES(nblBuiltinResourceData)
```

Comment:
why the extra macro?

Reply:
because this target must be created in this CMakeLists file

Reply:
then write it as a comment here and at macro declaration

Reply:
corrected in following commit